Doc/howto/logging.rst

```diff
@@ -141,6 +149,7 @@ messages:
    DEBUG:root:This message should go to the log file
    INFO:root:So should this
    WARNING:root:And this, too
+   ERROR:root:And non-ASCII stuff, too, like Øresund and Malmö
```

Comment:
We've had problems in the past with non-ASCII characters in the docs (breaking the CHM build in particular). I don't remember exactly how we dealt with it, but we might need some kind of escape (or we may have it already...)

Reply:
The [Unicode HOWTO](https://raw.githubusercontent.com/python/cpython/3.7/Doc/howto/unicode.rst) appears to contain Unicode characters without any special escaping: for example,

   265E    '♞': BLACK CHESS KNIGHT
   265F    '♟': BLACK CHESS PAWN

Reply:
You're right, and they're rendering properly, so I guess it's fine.